libs/core/src/test/java/org/elasticsearch/common/util/ESSloppyMathTests.java

```diff
@@ -19,6 +20,8 @@ public class ESSloppyMathTests extends ESTestCase {
     static double ATAN_DELTA = 1E-15;
     // accuracy for sinh(x)
     static double SINH_DELTA = 1E-15; // for small x
+    // accuracy for log(x)
+    static double LOG_DELTA = 1E-12;
```

Comment:
Since this is a lower precision than elsewhere I wondered if it would have consequences. But I noticed that the y calculation uses log directly, so it seems the error on y should be exactly the error on log, and I presume 1E-12 is good enough for any real-world accuracy for latitude?

Reply:
Yes, I think the accuracy is more than enough for us